docs/content/docs/connectors/datastream/hybridsource.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Hybrid Source
+weight: 8
+type: docs
+aliases:
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Hybrid Source
+
+`HybridSource` is a source that contains a list of concrete sources.
```

Comment:
Is it better to explain that this Source is a better point based on FLIP-27?

Reply:
I'd not explicitly mention FLIP-27. Eventually source=FLIP-27, if we do a good enough job. But maybe we could link to https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/datastream/sources/?

Reply:
+1, added the link